<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageEdit</class>
 <widget class="QMainWindow" name="ImageEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageEdit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>300</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>Pointer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>300</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>Pointer2</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>480</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Apply">
    <property name="geometry">
     <rect>
      <x>604</x>
      <y>440</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BlurButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Blur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SharpenButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sharpen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BlackWhiteButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Black&amp;White</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BrightnessButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>440</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Brightnesslvl">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>480</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimum">
     <number>-255</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ContrastButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrast</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Contrastlvl">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>480</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="NegativeButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>440</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Negative</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopButton">
    <property name="geometry">
     <rect>
      <x>604</x>
      <y>510</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GWButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&quot;Gray world&quot;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StampingButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>480</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stamping</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WaterColorButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>480</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>WaterColor</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
